packages/launchpad/src/Main.vue

```diff
@@ -150,8 +150,10 @@ const mutation = useMutation(Main_ResetErrorsAndLoadConfigDocument)
 
 const resetErrorAndLoadConfig = (id: string) => {
   if (!mutation.fetching.value) {
-    mutation.executeMutation({ id })
+    return mutation.executeMutation({ id })
   }
+
+  return
 }
 const query = useQuery({ query: MainLaunchpadQueryDocument })
```

Comment:
Cool - functionality seems good, I think this might be a chance to reduce some of the debt incurred (mainly around non idiomatic Vue code) introduced over the last little while. Thoughts?

Also, wonder if we can get a test for this. The refactor I suggested would let us easily unit test the BaseError as a dumb/presentational component, and e2e test the whole workflow.

We should be able to test this, maybe we can pair on it? I will think about some testing strategies in the meantime.